dataframe/dataframe.go

```diff
@@ -1845,3 +1883,7 @@ type Matrix interface {
 	Dims() (r, c int)
 	At(i, j int) float64
 }
+
+func createErr(cause, msg string, a ...interface{}) error {
+	return errors.Wrap(fmt.Errorf(msg, a...), cause)
+}
```

Comment:
Better, but still not a fan of this construct. And I really think you are over-using the wrapped error. It's a good paradigm, but doesn't make sense in all cases (as with the `Dataframe.Err` context or when you actually only want a `string` instead of an error.